<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CheckButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Чеки</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SortBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SortButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>99</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GoodsButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Товары</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShopButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>11</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Магазины</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FiltButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>47</y>
      <width>99</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Фильтровать</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>761</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="FiltBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
